Makefile

```diff
@@ -408,4 +410,6 @@ help: ## show this help screen
 		awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
 	@echo ''
 
+funcgen:
```

Comment:
This target is "PHONY" instead of being a proper make target that is named for the output file and has input file dependencies... That way it will correctly re-run only when necessary.